src/cls/IPM/Main.cls

```diff
@@ -2026,7 +2032,13 @@ ClassMethod Unpublish(ByRef pCommandInfo) [ Internal ]
 	Try {
 		Set tManager = ##class(%IPM.Repo.Remote.PublishManager).%Get(.tSC)
```

Comment:
To keep this consistent, I think the new repo should also have its own PublishManager and then only instantitate the publish manager based on the type value returned above (can instantiate it when setting tServer itself.)

Reply:
Done.